doc/api/dgram.md

```diff
@@ -154,6 +160,8 @@ Returns an object containing the address information for a socket.
 For UDP sockets, this object will contain `address`, `family` and `port`
 properties.
 
+This method throws `EBADF` if called on an unbound socket.
```

Comment:
```suggestion
This method throws an `Error` with code `EBADF` if called on an unbound socket.
```

Reply:
I think these should be wrapped into an appropriate Node error anyway. Leaving as is for now.